sources/packages/backend/apps/api/src/route-controllers/user/user.aest.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { Controller, Get, Put } from "@nestjs/common";
+import { UserService } from "../../services";
+import BaseController from "../BaseController";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import { BCeIDAccountsAPIOutDTO } from "./models/bceid-accounts.dto";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { ApiTags, ApiUnprocessableEntityResponse } from "@nestjs/swagger";
+import { UserControllerService } from "..";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Controller("users")
+@ApiTags("users")
+export class UserAESTController extends BaseController {
+  constructor(
+    private readonly service: UserService,
+    private readonly userControllerService: UserControllerService,
+  ) {
+    super();
+  }
+
+  @ApiUnprocessableEntityResponse({
+    description:
+      "Not able to retrieve BCeID business account details for the current authenticated user.",
+  })
+  @Get("bceid-accounts")
+  async getAllBCeIDs(
+    @UserToken() userToken: IUserToken,
+  ): Promise<BCeIDAccountsAPIOutDTO> {
+    return await this.userControllerService.getAllBCeIDs(userToken);
```

Comment:
The `await` can be removed here.

Reply:
I was not able to see this change. Is it updated?

Reply:
Yes, please see https://github.com/bcgov/SIMS/pull/1627#pullrequestreview-1231041755

Reply:
Wrong file same issue. The file `user.institutions.controller.ts` has the same.
![image](https://user-images.githubusercontent.com/61259237/209729420-dc935280-c9b5-488d-9150-4e177e087649.png)

Reply:
ok. Removed as well.